src/Microsoft.AspNet.Mvc.Core/FileStreamResult.cs

```diff
@@ -13,48 +13,48 @@ namespace Microsoft.AspNet.Mvc
 {
     /// <summary>
     /// Represents an <see cref="ActionResult"/> that when executed will
-    /// write a file from a stream to the response.
+    /// write a stream to the response.
     /// </summary>
-    public class FileStreamResult : FileResult
+    public class StreamResult : FileResult
```

Comment:
I think this name is confusing...for users of Web API they might translate `StreamContent` -> `StreamResult` just like how `ObjectContent` became `ObjectResult`